services/wireguard/service/service_unix.go

```diff
@@ -175,7 +180,17 @@ func (m *Manager) ProvideConfig(sessionConfig json.RawMessage) (*session.ConfigP
 		return nil, errors.Wrap(err, "failed to setup NAT/firewall rules")
 	}
 
+	statsPublisher := newStatsPublisher(m.publisher, 3*time.Second)
```

Comment:
In https://github.com/mysteriumnetwork/node/pull/1708 stats update interval is 2 seconds. Not sure if this is related.

Reply:
We can have more frequent emitting on consumer side, as that is only tracking a single session. The more frequent the interval, the more accurate the payments.